code/modules/paperwork/adminpaper.dm

```diff
@@ -0,0 +1,121 @@
+//Adminpaper - it's like paper, but more adminny! Used for sending faxes.
+/obj/item/weapon/paper/admin
```

Comment:
Make a datum that formats a piece of paper correctly instead of having a special piece of paper